api-samples/tabCapture/background.js

```diff
@@ -0,0 +1,31 @@
+let currentReceiverTabId = null;
+
+chrome.action.onClicked.addListener(async (tab) => {
+  const currentTabId = tab.id;
+
+  if (currentReceiverTabId) {
```

Comment:
If the service worker gets terminated for being idle, this will be lost. Could we instead do `chrome.tabs.query` and search for any tabs with the URL?